### PR TITLE
Vulnerability scanner workaround

### PR DESCRIPTION
**Pull request by kasiakepka**
**Description**

Changes proposed in this pull request:

- Implement workaround so that vulnerability scanners does not fail

**Related issue(s)**

https://github.com/kyma-project/test-infra/issues/1705

### CODE REVIEW

File: development/vulnerabilities-scanner.sh [hunk 1]
```diff
@@ -105,20 +105,24 @@ function testComponents() {
...
     GOPKG_FILE_NAME="${DIR}"Gopkg.lock
 
     if [ -f "${GOPKG_FILE_NAME}" ]; then
+      set +e
       # fetch dependencies
       echo " ├── fetching dependencies..."
       cd "${DIR}"
       dep ensure --vendor-only
+      if [[ $? != 0 ]]; then
+        echo -e "\n-----------------------------------------------\nClould not test ${DIR}"
```
Comment (antoineco) [inline development/vulnerabilities-scanner.sh]: "Could"

File: development/vulnerabilities-scanner.sh [hunk 2]
```diff
@@ -105,20 +105,24 @@ function testComponents() {
   do
 
     echo "processing ${DIR}"
-
+    
     GOPKG_FILE_NAME="${DIR}"Gopkg.lock
```
Comment (antoineco) [inline development/vulnerabilities-scanner.sh]: Oops, leading spaces.

File: development/vulnerabilities-scanner.sh [hunk 3]
```diff
@@ -105,20 +105,24 @@ function testComponents() {
...
 
     if [ -f "${GOPKG_FILE_NAME}" ]; then
+      set +e
```
Comment (antoineco) [inline development/vulnerabilities-scanner.sh]: `dep ensure || dep ensure` would be a simple way to achieve the same result with setting that option.

**Comment by kasiakepka** [comment]
The workaround is no longer necessary as scan is working fine. We were not able able to identify what exactly fixed the issue however it happened after https://github.com/kyma-project/kyma/pull/6272

**Pull request closed without merging**